sim/shaman/lightning_shield.go

```diff
@@ -66,10 +69,14 @@ func (shaman *Shaman) registerLightningShieldSpell() *core.Spell {
 			},
 		},
 		ApplyEffects: func(sim *core.Simulation, _ *core.Unit, _ *core.Spell) {
-			shaman.LightningShieldAura.Activate(sim)
-			shaman.LightningShieldAura.SetStacks(sim, 3+(2*shaman.Talents.StaticShock))
+			shaman.activateLightningShieldAura(sim)
 		},
 	})
 
 	return (shaman.LightningShield)
 }
+
+func (shaman *Shaman) activateLightningShieldAura(sim *core.Simulation) {
+	shaman.LightningShieldAura.Activate(sim)
+	shaman.LightningShieldAura.SetStacks(sim, defaultLightningSheildStacks+(staticShockStackMultiplier*shaman.Talents.StaticShock))
```

Comment:
same typo here, also im not sure that replacing "2" with "staticShockStackMultiplier" helps clarity very much, also inconsistent with most other spells